packages/indexer-agent/src/indexer.ts

```diff
@@ -332,6 +335,105 @@ export class Indexer {
     }
   }
 
+  async costModels(): Promise<CostModelAttributes[]> {
+    try {
+      const result = await this.indexerManagement
+        .query(
+          gql`
+            query costModels {
+              costModels {
+                deployment
+                model
+                variables
+              }
+            }
+          `,
+        )
+        .toPromise()
+
+      if (result.error) {
+        throw result.error
+      }
+      this.logger.debug('Fetched cost models', {
+        count: result.data.costModels.length,
+        model: result.data.costModels.map((rule: CostModelAttributes) => {
+          return {
+            identifier: rule.deployment,
+            model: rule.model,
+            variables: rule.variables,
+          }
+        }),
+      })
+      return result.data.costModels
+    } catch (error) {
+      this.logger.error(`Failed to query indexer management API`, { error })
+      throw error
+    }
+  }
+
+  async ensureGlobalCostModel(): Promise<void> {
```

Comment:
Not sure if this is a good idea (Should I remove `basic.agora` in `indexer-agent` package?) since now indexer could `set model global __` and `set variable global __` through cli

Reply:
Good question, let's discuss next time we meet.

Reply:
I think we can remove the default cost model file right now, and simply provide the easy way for indexers to set their default cost model.  That way we aren't defining a default across all indexers that is dependent on their indexer software version.